src/controllers/auth.js

```diff
@@ -1,24 +1,25 @@
-import { config } from 'dotenv';
-import userServices from '../services/user';
+import UserService from '../services/user.service';
 import successRes from '../utils/successResponse';
 import ErrorResponse from '../utils/errorResponse';
 import encryption from '../helpers/encryption';
 
-config();
-
 const duration = process.env.TOKEN_EXPIRE;
 
-const { findUser } = userServices;
+const { findUser } = UserService;
 const { signToken, checkPassword } = encryption;
 
 class Auth {
   static async login(req, res, next) {
     try {
       const { email, password } = req.body;
+
       const foundUser = await findUser({ email });
-      if (!foundUser) return ErrorResponse(res, 404, 'Invalid credentials');
```

Comment:
This check is very important because it will fail on the next line if found user is null

Reply:
The findUser function is set to throw an error where in this case it will send that error to the response